packages/template-retail-react-app/app/pages/checkout/partials/shipping-address-selection.jsx

```diff
@@ -38,7 +38,8 @@ const ShippingAddressEditForm = ({
     toggleAddressEdit,
     hideSubmitButton,
     form,
-    submitButtonLabel
+    submitButtonLabel,
+    ariaLabel
```

Comment:
NIT: this prop name is a bit generic, and there are many elements in this component, we may want to name it a bit specific to know where it belongs to

Reply:
I thought of naming as addressAriaLabel but the component itself is about address so it is kind of repetitive. The two main component it has ShippingAddressEditForm and ShippingAddressSelection. And both need this label